src/containers/Tenant/Diagnostics/TenantOverview/TenantDashboard/TenantDashboard.tsx

```diff
@@ -26,16 +27,52 @@ export interface ChartConfig {
 
 interface TenantDashboardProps {
     charts: ChartConfig[];
+    onDashboardLoad?: VoidFunction;
 }
 
-export const TenantDashboard = ({charts}: TenantDashboardProps) => {
+type TenantDashboardStatus = 'success' | 'error' | undefined;
+
+type DashboardsChartsStatuses = Record<string, ChartDataStatus>;
+
+export const TenantDashboard = ({charts, onDashboardLoad}: TenantDashboardProps) => {
+    const [dashboardStatus, setDashboardStatus] = useState<TenantDashboardStatus>();
+    const chartsStatuses = useRef<DashboardsChartsStatuses>({});
```

Comment:
It seem a bit redundant: you need to know if at least one chart is succeed. So, no need to check charts id's:
```
const [showDashboard, setShowDashboard] = useState(false);
... one chart was loaded => setShowDashboard(true) ...
... no chats was loaded => do nothing ...
if (!showDashboard) {
  return null;
}
```

Reply:
Yep, updated it. Didn't figured out in the beginning, that we can prevent charts refetches with autorefresh variable